packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -258,11 +262,11 @@ export const renderVideoEpisodes = ({
               </VisuallyHiddenText>
             </EpisodeList.Link>
             {episode.episodeTitle && (
-              <span role="text">
-                <EpisodeList.Metadata as="time">
+              <div>
```

Comment:
Do you think this change should also be carried forward to Simorgh @jroebu14 ?

- The `div` turns it into a block element, which means when css is disabled, it still correctly breaks onto the next line
- `role=text` is unnecessary because there is only one nested text element. I think this is also true in Simorgh?

Reply:
Mocked the change in Simorgh regardless https://github.com/bbc/simorgh/pull/8602

Reply:
yep good shout as long as it's still OK with CSS enabled too!

Reply:
Nice one man - aye looks visually unchanged w/CSS. I've undrafted [the Simorgh PR](https://github.com/bbc/simorgh/pull/8602)